src/Grpc.NetCore.HttpClient/Grpc.NetCore.HttpClient.csproj

```diff
@@ -12,15 +12,14 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
Trailing headers is netcoreapp3.0 only today. I have an issue on dotnet/standard that proposes adding the property to netstandard2.1

Reply:
nit: add a TODO?